src/utilities/currency/index.ts

```diff
@@ -0,0 +1,10 @@
+export function format(value: number, precision: number = 2): string {
```

Comment:
Instead of putting directly `precision` as the second argument, I think is better to pass an object with `precision` inside. This makes the API more flexible for future features (maybe could happen).

Reply:
I agree that it'll let the function more flexible, I'll work on that change! :D